src/model/cloud-runner/services/core/shared-workspace-locking.ts

```
@@ -1,23 +1,75 @@
     return `${SharedWorkspaceLocking.workspaceBucketRoot}locks/`;
   }
+  private static get workspacePrefix() {
+    return `locks/`;
+  }
+  private static async ensureBucketExists(): Promise<void> {
+    const bucket = SharedWorkspaceLocking.bucket;
+    try {
+      await SharedWorkspaceLocking.s3.send(new HeadBucketCommand({ Bucket: bucket }));
+    } catch {
+      const region = Input.region || process.env.AWS_REGION || process.env.AWS_DEFAULT_REGION || 'us-east-1';
+      const createParams: any = { Bucket: bucket };
```

ESLint

[error] The variable `createParams` should be named `createParameters`. A more descriptive name will do too.

(unicorn/prevent-abbreviations)